intro-to-tests/practice-intro-to-pytest.md

```diff
@@ -0,0 +1,890 @@
+# Practice: Intro to Pytest
+
+## Directions
+
+Complete all questions below.
+
+## Practice
+
+### Determining Best Test Cases
+
+<!-- Question 1 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: multiple-choice
+* id: oLlaji
+* title: Intro to Pytest
+##### !question
+
+```python
+def calculate(num_a, num_b, operator):
+    if (isinstance(num_a, int) or isinstance(num_a, float)) and (isinstance(num_b, int) or isinstance(num_b, float)):
```

Comment:
I mentioned this in another worksheet PR, but duplicating the code head of each question may encourage students to spend time looking for minute differences rather than focusing on the actual questions.

Reply:
LOOOOOOOL. Just gonna do a blanket "LOL" at myself with this one. :) Thanks for the catch

Reply:
@anselrognlie 

Weirdly enough, with your code, it also doesn't work as expected

![Screen Shot 2021-01-15 at 1 47 11 PM](https://user-images.githubusercontent.com/30602862/104781897-2d9d4a00-5738-11eb-9df9-91911dd1dbad.png)


I just rethought through it, should it be:

```python
   if (
       not (isinstance(num_a, int)) or
       not (isinstance(num_a, float)) or
       not (isinstance(num_b, int)) or
       not (isinstance(num_b, float))
   ):
       return "Type Error: num_a and num_b must be integer or float"
```

(i'll look up the pythonic way to do this formatting)

Reply:
Sorry. I assumed a parenthesization that wasn't there.  I just meant to use De Morgans law to invert the comparison, since the existing code was checking for int or float for each value.  Just checked this in actual code. :)

```python
    if (not ((isinstance(num_a, int) or isinstance(num_a, float)) and (isinstance(num_b, int) or isinstance(num_b, float)))):
```

Reply:
maybe formatted as
```python
    if not (
        (isinstance(num_a, int) or isinstance(num_a, float)) and 
        (isinstance(num_b, int) or isinstance(num_b, float))):

```

Reply:
Love a well-placed negation.

Funnily enough, my formatter wants to format it as:

```suggestion
    if not (
        (isinstance(num_a, int) or isinstance(num_a, float)) and
            (isinstance(num_b, int) or isinstance(num_b, float))):
```

which is pretty funny/interesting

Reply:
That makes my heart weep. 😰